packages/api-rx/src/types.d.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017-2018 @polkadot/api-rx authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { Observable } from 'rxjs';
+import { ExtrinsicFunction } from '@polkadot/extrinsics/types';
+import RpcRx from '@polkadot/rpc-rx/index';
```

Comment:
could move line 5 and 7 down since their non-type imports

Reply:
As indicated with polkadot/types the water is now _very_ muddied. Actually starting to believe to gain some clarity, that the types/package split needs to go. (Unless we pull some magic in /types, but that is not quite possible unless we start making extra interfaces for all)